qoffer-infrastructure/src/test/resources/DatabaseSchema/OfferDBConnectorSchema

```diff
@@ -0,0 +1,56 @@
+CREATE TABLE product (
+    id int(11) UNIQUE PRIMARY KEY,
+    category ENUM('Sequencing','Project Management','Primary Bioinformatics','Secondary Bioinformatics','Data Storage'),
+    description varchar(2500),
+    productName varchar(500),
+    unitPrice double(11,2),
+    unit varchar(10)
```

Comment:
We defined an enum in the DTO, I would expect a controlled vocab here as well. What do you think?

Reply:
When we have defined the values I can change it. I would do it when it is changed in the DB then at the same time.